ext/intl/calendar/calendar_methods.cpp

```diff
@@ -380,6 +380,7 @@ U_CFUNC PHP_FUNCTION(intlcal_before)
 U_CFUNC PHP_FUNCTION(intlcal_set)
 {
 	zend_long args[6];
+	zend_bool args_is_null[6];
```

Comment:
Doesn't this need some initialization like `= { 0, 0, 1, 1, 1, 1 }`?

Reply:
Yes, you were right. I forgot about initialization which caused some failed tests.